### ISSUE TITLE
Progress bar during addon-install task causes silent hang while waiting for user input.

### ISSUE THREAD

**Issue by kybishop** [ember-cli/ember-cli#6694]
The progress bar during the addon-install task hides output from `this.ui.prompt`. Pretty sure this is because `this.ui.prompt` does not insert a newline. As a result of the hidden output, any blueprint which invokes `this.ui.prompt` silently waits for input while the progress bar spins.

Offending line: https://github.com/ember-cli/ember-cli/blob/dacb706f8727b20726b337f66ea4cd45d0074278/lib/tasks/addon-install.js#L37

A particularly problematic prompt, which I don't believe users have control of, is file overwrites: https://github.com/ember-cli/ember-cli/blob/e38940e166435ebbc1ebbdc8ee49405b8b978946/lib/models/file-info.js#L67

---

Output from `ember version --verbose && npm --version`:
```
ember-cli: 2.9.1
http_parser: 2.7.0
node: 7.2.0
v8: 5.4.500.43
uv: 1.10.1
zlib: 1.2.8
ares: 1.10.1-DEV
modules: 51
openssl: 1.0.2j
icu: 58.1
unicode: 9.0
cldr: 30.0.2
tz: 2016g
os: darwin x64
3.10.9
```

**Comment by rwwagner90** [comment]
Anytime `addAddonToProject` is invoked, and the addon runs a blueprint to generate a file, and there is a prompt to overwrite the file or not, that prompt is hidden by the progress bar issue as well.

**Comment by stefanpenner** [comment]
I believe this may be addressed in master, now that `npm` and `bower` are invoked via `execa`.

cc @Turbo87

**Comment by Turbo87** [comment]
Yes, this seems to be a duplicate of https://github.com/ember-cli/ember-cli/issues/6593 and will be fixed by the planned v2.10.1 release tomorrow.
